portal/templates/portal_wrapper.html

```diff
@@ -86,6 +86,7 @@
                             {% if user and (user.has_role(ROLE.STAFF) or user.has_role(ROLE.INTERVENTION_STAFF)) %}
                             <li><a href="{{PORTAL}}/patients/">{{ _("Patients") }}</a></li>
                             {% endif %}
+                            {% if user and user.has_roles(ROLE.STAFF_ADMIN) %}<li><a href="{{PORTAL}}/staff">{{ _("Staff List") }}</a></li>{% endif %}
                             {% if user and user.has_roles('admin') %}
```

Comment:
It'd be nice to clean up all the outdated string comparisons - i.e. user.has_roles() should always take the respective ROLE.NAME rather than strings like "admin"